app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb

```diff
@@ -88,7 +88,7 @@ def get_hostname(virtual_machine)
     if vm_provider == "Amazon EC2"
       virtual_machine.dig("details", "tags.Name") || virtual_machine.dig("details", "tags", "Name")
     elsif vm_provider == "IBM"
-      virtual_machine.dig("details", "hostname")
+      virtual_machine.dig("details", "hostname") || virtual_machine.dig("details", "name")
```

Comment:
Can the hostname actually be in either of these places or you're just using the name as the hostname so it won't be `nil`?

Reply:
@agrare The hostnames are located in different field depending on which IBM Cloud infrastructure the vm was deployed to. (classic infrastructure(aka old SoftLayer) vs VPC)

Reply:
:+1:

Reply:
@k-chu can you add that to the PR description?